src/routes/terms/+page.svelte

```diff
@@ -1,42 +1,453 @@
+                                        <feFlood flood-opacity="0" result="BackgroundImageFix" />
+                                        <feGaussianBlur
+                                            in="BackgroundImageFix"
+                                            stdDeviation="100"
+                                        />
+                                        <feComposite
+                                            in2="SourceAlpha"
+                                            operator="in"
+                                            result="effect1_backgroundBlur_1684_10747"
+                                        />
+                                        <feBlend
+                                            mode="normal"
+                                            in="SourceGraphic"
+                                            in2="effect1_backgroundBlur_1684_10747"
+                                            result="shape"
+                                        />
+                                    </filter>
+                                    <linearGradient
+                                        id="paint0_linear_1684_10747"
+                                        x1="2.02105"
+                                        y1="1.10843"
+                                        x2="16.3872"
+                                        y2="17.2901"
+                                        gradientUnits="userSpaceOnUse"
+                                    >
+                                        <stop stop-color="white" stop-opacity="0.4" />
+                                        <stop offset="1" stop-color="white" stop-opacity="0" />
+                                    </linearGradient>
+                                    <linearGradient
+                                        id="paint1_linear_1684_10747"
+                                        x1="7.45643"
+                                        y1="-1.10615"
+                                        x2="5.53812"
+                                        y2="17.9973"
+                                        gradientUnits="userSpaceOnUse"
+                                    >
+                                        <stop stop-color="white" stop-opacity="0.16" />
+                                        <stop offset="1" stop-color="white" stop-opacity="0" />
+                                    </linearGradient>
+                                    <clipPath id="clip0_1684_10747">
+                                        <rect width="16" height="16" fill="white" />
+                                    </clipPath>
+                                </defs>
+                            </svg>
+                        </button>
+                    </div>
+                    <ul class="aw-page-steps-list aw-sub-body-500">
+                        <li>
+                            <a href="#introduction" class="is-selected">Introduction</a>
```

Comment:
This will always be selected this way, but then I noticed privacy is also this way... Will create an Asana task so I can fix it, you can leave it as is